website/content/index.mdx

```diff
@@ -0,0 +1,202 @@
+- **Advanced batch scheduling**: Fair queuing, gang scheduling, preemption, and resource limits
+- **Enterprise-grade reliability**: Secure, highly available components designed for production use
+
+As a [CNCF Sandbox project](https://www.cncf.io/), Armada is actively maintained and used in production environments, including at [G-Research](https://www.gresearch.com/) where it processes millions of jobs daily.
+
+## Why Use Armada?
+
+### Kubernetes Limitations for Batch Workloads
+
+Traditional Kubernetes faces several challenges when running batch workloads at scale:
+
+1. **Single Cluster Scaling Limits**: Scaling a single Kubernetes cluster beyond a certain size is [challenging](https://openai.com/blog/scaling-kubernetes-to-7500-nodes/), typically maxing out around 5,000-15,000 nodes depending on configuration.
+
+2. **Storage Backend Constraints**: Achieving very high throughput using etcd, Kubernetes' in-cluster storage backend, is [challenging](https://etcd.io/docs/v3.5/op-guide/performance/) and can become a bottleneck for job queuing.
+
+3. **Inadequate Batch Scheduling**: The default [kube-scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/) lacks essential batch scheduling features like fair queuing, gang scheduling, and intelligent preemption.
+
+### Armada's Solution
+
+Armada overcomes these limitations by:
+
+- **Distributing across multiple clusters**: Manage thousands of nodes across many Kubernetes clusters
+- **Out-of-cluster scheduling**: Perform queueing and scheduling using specialized storage layers
+- **Purpose-built batch scheduler**: Include advanced scheduling features designed specifically for batch workloads
+
+## Key Features and Benefits
+
+### Core Scheduling Features
+
+**Fair-Use Scheduling**
+
+- Maintains fair resource share over time across users and teams
+- Based on dominant resource fairness principles
+- Includes priority factors for different queues
+- Inspired by HTCondor priority systems
+
+**High Throughput Processing**
+
+- Handle millions of queued jobs simultaneously
+- Specialized storage layer optimized for batch workloads
+- Efficient job submission and status tracking
+
+**Gang Scheduling**
+
+- Atomically schedule sets of related jobs
+- Ensures all jobs in a group start together or not at all
+- Critical for distributed computing frameworks like MPI
+
+**Intelligent Preemption**
+
+- Run urgent jobs in a timely fashion
+- Balance resource allocation between users
+- Configurable preemption policies
+
+### Enterprise-Grade Operations
+
+**Massive Scale Support**
+
+- Utilize multiple Kubernetes clusters simultaneously
+- Scale beyond single cluster limitations
+- Add and remove clusters without service disruption
+
+**Advanced Resource Management**
+
+- Resource and job scheduling rate limits
+- Detailed resource allocation controls
+- Node affinity and anti-affinity rules
```

Comment:
This is a Kubernetes concept, not an Armada concept, so I'd remove this

Reply:
Removed